stories/ClaimSection.stories.tsx

```diff
@@ -13,6 +14,6 @@ const Template: Story<ClaimSectionProps> = (args) => <ClaimSectionComponent {...
 
 export const ClaimSection = Template.bind({})
 ClaimSection.args = {
-  ...ClaimDetailsStories.ClaimDetails.args,
-  ...ClaimStatusStories.ClaimStatus.args,
+  detailsContent: ClaimDetailsStories.ClaimDetails.args as ClaimDetailsContent,
+  statusContent: ClaimStatusStories.ClaimStatus.args as ClaimStatusContent,
```

Comment:
Note: For stories, I've put test/dummy values into the child component and then referenced those values in parent components. In this case, the default dummy values are in `ClaimDetails.stories.tsx` and they are pulled into `ClaimSection.stories.tsx` here.